### ISSUE TITLE
vector upload error quiries

### ISSUE THREAD

**Issue by Nagarjuna-jantali** [usnistgov/ACVP-Server#96]
Hello,
 i am trying to upload the rsp file for validation to nist demo server, but getting below error, please let me know what it is and how can i resolve this

./app/acvp_app --ecdsa  --vector_upload ecdsa_rsp.json

***ACVP [STATUS][acvp_upload_vectors_from_file:1109]--> Uploading vectors from response file...
***ACVP [STATUS][acvp_upload_vectors_from_file:1242]--> Sending responses for vector set 525659
***ACVP [WARN][execute_network_action:1108]--> JWT authorization has timed out, curl rc=401. Refreshing session...
***ACVP [STATUS][acvp_login:2676]--> Logging in...
***ACVP [STATUS][acvp_login:2696]--> Login successful
***ACVP [STATUS][execute_network_action:1114]--> Refresh successful, attempting to continue...
***ACVP [ERR][execute_network_action:1163]--> Refreshed + retried, HTTP transport fails. curl rc=400

***ACVP [ERR][log_network_status:1247]--> 400 error received from server. Message:
***ACVP [ERR][log_network_status:1248]--> [
  {
    "acvVersion": "1.0"
  },
  {
    **"error": "Validation error(s) on JSON payload.",
    "context": [
      "Vector set not in \u0027failed\u0027 status.",
      "Unable to submit results when vector set in \u0027passed\u0027 status."**
    ]
  }
]
***ACVP [ERR][acvp_upload_vectors_from_file:1245]--> Failed to submit test results for vector set - skipping...
***ACVP [STATUS][acvp_upload_vectors_from_file:1242]--> Sending responses for vector set 525660
***ACVP [ERR][log_network_status:1247]--> 400 error received from server. Message:
***ACVP [ERR][log_network_status:1248]--> [
  {
    "acvVersion": "1.0"
  },
  {
    "error": "Validation error(s) on JSON payload.",
    "context": [
      "Vector set not in \u0027failed\u0027 status.",
      "Unable to submit results when vector set in \u0027passed\u0027 status."
    ]
  }
]
***ACVP [ERR][acvp_upload_vectors_from_file:1245]--> Failed to submit test results for vector set - skipping...
***ACVP [STATUS][acvp_upload_vectors_from_file:1242]--> Sending responses for vector set 525661
***ACVP [ERR][log_network_status:1247]--> 400 error received from server. Message:
***ACVP [ERR][log_network_status:1248]--> [
  {
    "acvVersion": "1.0"
  },
  {
    "error": "Validation error(s) on JSON payload.",
    "context": [
      "Vector set not in \u0027failed\u0027 status.",
      "Unable to submit results when vector set in \u0027passed\u0027 status."
    ]
  }
]
***ACVP [ERR][acvp_upload_vectors_from_file:1245]--> Failed to submit test results for vector set - skipping...
***ACVP [STATUS][acvp_upload_vectors_from_file:1242]--> Sending responses for vector set 525662
***ACVP [ERR][log_network_status:1247]--> 400 error received from server. Message:
***ACVP [ERR][log_network_status:1248]--> [
  {
    "acvVersion": "1.0"
  },
  {
    "error": "Validation error(s) on JSON payload.",
    "context": [
      "Vector set not in \u0027failed\u0027 status.",
      "Unable to submit results when vector set in \u0027passed\u0027 status."
    ]
  }
]
***ACVP [ERR][acvp_upload_vectors_from_file:1245]--> Failed to submit test results for vector set - skipping...
***ACVP [STATUS][acvp_upload_vectors_from_file:1258]--> Tests complete, checking results...
***ACVP [STATUS][acvp_get_result_test_session:3054]--> Passed all vectors in test session!

Thanks,
Nag

**Comment by Kritner** [comment]
Is there a reason you're attempting to upload a response for a vector sets that are already in a passed state?  As the message in the server response indicates, we don't accept a "new upload" of responses if the vector set is already passed.

**Comment by Nagarjuna-jantali** [comment]
> Is there a reason you're attempting to upload a response for a vector sets that are already in a passed state? As the message in the server response indicates, we don't accept a "new upload" of responses if the vector set is already passed.

Hello,

I was trying with openssl library in build machine, now i really want to test it on actual host machine openssl library . Is there any way we can  try new upload ?

Thanks,
Nag

**Comment by Kritner** [comment]
> I believe i need to download new request files , that potentially create new session and can use this new request file to generate response and than upload , is this right ?

That is correct. If you're literally just running the same prompt files across multiple different machines, then could you just compare your produced response files between machines, at least for several algorithms.

I suppose from that perspective it could be useful to upload a response file to a vector set that is already in the passed state, I'm not sure if anyone has inquired about that previously.

It's not necessarily something we *couldn't* allow for, we just don't currently.
